Jenkinsfile_CNP

```diff
@@ -41,6 +41,8 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
+GradleBuilder builder = new GradleBuilder(this, product)
```

Comment:
requires import as well https://github.com/hmcts/bulk-scan-processor/blob/master/Jenkinsfile_CNP#L5

Reply:
Thanks @timja - we're still getting the same error of..

ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.